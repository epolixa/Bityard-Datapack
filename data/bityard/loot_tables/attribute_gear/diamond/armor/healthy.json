{
  "pools": [
    {
      "rolls": 1,
      "entries": [
        {
          "type": "item",
          "name": "minecraft:diamond_helmet",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Healthy Diamond Helmet\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.armor",
                  "name": "Armor",
                  "amount": 3.0,
                  "operation": "addition",
                  "slot": "head"
                },
                {
                  "attribute": "generic.armorToughness",
                  "name": "Armor Toughness",
                  "amount": 2.0,
                  "operation": "addition",
                  "slot": "head"
                },
                {
                  "attribute": "generic.maxHealth",
                  "name": "Max Health",
                  "amount": 2.0,
                  "operation": "addition",
                  "slot": "head"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:diamond_chestplate",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Healthy Diamond Chestplate\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.armor",
                  "name": "Armor",
                  "amount": 8.0,
                  "operation": "addition",
                  "slot": "chest"
                },
                {
                  "attribute": "generic.armorToughness",
                  "name": "Armor Toughness",
                  "amount": 2.0,
                  "operation": "addition",
                  "slot": "chest"
                },
                {
                  "attribute": "generic.maxHealth",
                  "name": "Max Health",
                  "amount": 2.0,
                  "operation": "addition",
                  "slot": "chest"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:diamond_leggings",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Healthy Diamond Leggings\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.armor",
                  "name": "Armor",
                  "amount": 6.0,
                  "operation": "addition",
                  "slot": "legs"
                },
                {
                  "attribute": "generic.armorToughness",
                  "name": "Armor Toughness",
                  "amount": 2.0,
                  "operation": "addition",
                  "slot": "legs"
                },
                {
                  "attribute": "generic.maxHealth",
                  "name": "Max Health",
                  "amount": 2.0,
                  "operation": "addition",
                  "slot": "legs"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:diamond_boots",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Healthy Diamond Boots\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.armor",
                  "name": "Armor",
                  "amount": 3.0,
                  "operation": "addition",
                  "slot": "feet"
                },
                {
                  "attribute": "generic.armorToughness",
                  "name": "Armor Toughness",
                  "amount": 2.0,
                  "operation": "addition",
                  "slot": "feet"
                },
                {
                  "attribute": "generic.maxHealth",
                  "name": "Max Health",
                  "amount": 2.0,
                  "operation": "addition",
                  "slot": "feet"
                }
              ]
            }
          ]
        }
      ]
    }
  ]
}