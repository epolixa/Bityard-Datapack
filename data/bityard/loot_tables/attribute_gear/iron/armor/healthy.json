{
  "pools": [
    {
      "rolls": 1,
      "entries": [
        {
          "type": "item",
          "name": "minecraft:iron_helmet",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{loot_type:\"attribute_gear\", display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Healthy Iron Helmet\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.armor",
                  "name": "Armor",
                  "amount": 2.0,
                  "operation": "addition",
                  "slot": "head"
                },
                {
                  "attribute": "generic.maxHealth",
                  "name": "Max Health",
                  "amount": 2.0,
                  "operation": "addition",
                  "slot": "head"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:iron_chestplate",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{loot_type:\"attribute_gear\", display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Healthy Iron Chestplate\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.armor",
                  "name": "Armor",
                  "amount": 6.0,
                  "operation": "addition",
                  "slot": "chest"
                },
                {
                  "attribute": "generic.maxHealth",
                  "name": "Max Health",
                  "amount": 2.0,
                  "operation": "addition",
                  "slot": "chest"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:iron_leggings",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{loot_type:\"attribute_gear\", display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Healthy Iron Leggings\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.armor",
                  "name": "Armor",
                  "amount": 5.0,
                  "operation": "addition",
                  "slot": "legs"
                },
                {
                  "attribute": "generic.maxHealth",
                  "name": "Max Health",
                  "amount": 2.0,
                  "operation": "addition",
                  "slot": "legs"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:iron_boots",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{loot_type:\"attribute_gear\", display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Healthy Iron Boots\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.armor",
                  "name": "Armor",
                  "amount": 2.0,
                  "operation": "addition",
                  "slot": "feet"
                },
                {
                  "attribute": "generic.maxHealth",
                  "name": "Max Health",
                  "amount": 2.0,
                  "operation": "addition",
                  "slot": "feet"
                }
              ]
            }
          ]
        }
      ]
    }
  ]
}