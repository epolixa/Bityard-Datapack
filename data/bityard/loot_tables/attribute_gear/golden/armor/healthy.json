{
  "pools": [
    {
      "rolls": 1,
      "entries": [
        {
          "type": "item",
          "name": "minecraft:golden_helmet",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{loot_type:\"attribute_gear\", display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Healthy Golden Helmet\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.armor",
                  "name": "Armor",
                  "amount": 2.0,
                  "operation": "addition",
                  "slot": "head"
                },
                {
                  "attribute": "generic.maxHealth",
                  "name": "Max Health",
                  "amount": 2.0,
                  "operation": "addition",
                  "slot": "head"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:golden_chestplate",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{loot_type:\"attribute_gear\", display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Healthy Golden Chestplate\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.armor",
                  "name": "Armor",
                  "amount": 5.0,
                  "operation": "addition",
                  "slot": "chest"
                },
                {
                  "attribute": "generic.maxHealth",
                  "name": "Max Health",
                  "amount": 2.0,
                  "operation": "addition",
                  "slot": "chest"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:golden_leggings",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{loot_type:\"attribute_gear\", display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Healthy Golden Leggings\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.armor",
                  "name": "Armor",
                  "amount": 3.0,
                  "operation": "addition",
                  "slot": "legs"
                },
                {
                  "attribute": "generic.maxHealth",
                  "name": "Max Health",
                  "amount": 2.0,
                  "operation": "addition",
                  "slot": "legs"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:golden_boots",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{loot_type:\"attribute_gear\", display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Healthy Golden Boots\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.armor",
                  "name": "Armor",
                  "amount": 1.0,
                  "operation": "addition",
                  "slot": "feet"
                },
                {
                  "attribute": "generic.maxHealth",
                  "name": "Max Health",
                  "amount": 2.0,
                  "operation": "addition",
                  "slot": "feet"
                }
              ]
            }
          ]
        }
      ]
    }
  ]
}